linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala

```diff
@@ -101,26 +107,39 @@ trait RouterConfig {
   var baseDtab: Option[Dtab] = None
   var failFast: Option[Boolean] = None
   var timeoutMs: Option[Int] = None
+  var dstPrefix: Option[String] = None
```

Comment:
This is unrelated to this review but I just thought of it.  If linker-level dtabs are gone and now dtabs are per-router, is a per-router `dstPrefix` still useful at all?